jest.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 	// Automatically clear mock calls and instances between every test
 	clearMocks: true,
 	preset: './gutenberg/node_modules/react-native/jest-preset.js',
-	setupFiles: [ '<rootDir>/' + configPath + '/setup.js' ],
+	setupFiles: [ '<rootDir>/' + configPath + '/setup.js', './jest_setup.js' ],
```

Comment:
Nitpick: I'm wondering if we could use a similar name format for `jest_setup.js` as this file, i.e. `jest.setup.js`, WDYT?

Reply:
Sure! I'll update it 👍

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5544/commits/9278461c173409d342520b8d7884c7f27b8f04cd and https://github.com/wordpress-mobile/gutenberg-mobile/pull/5544/commits/be3b94f1f4f716e706cee3dbbedb57d9647b5335